Makefile

```diff
@@ -107,13 +107,18 @@ MARCHFLAG.oneAPI        := $(MARCHFLAG.clang)
 OMP_SIMD_FLAG.gcc       := -fopenmp-simd
 OMP_SIMD_FLAG.clang     := $(OMP_SIMD_FLAG.gcc)
 OMP_SIMD_FLAG.icc       := -qopenmp-simd
-OMP_SIMD_FLAG.oneAPI    := $(OMP_SIMD_FLAG.clang)
+OMP_SIMD_FLAG.oneAPI    := $(OMP_SIMD_FLAG.icc)
```

Comment:
`icx` supports both.
```
  -fopenmp                Parse OpenMP pragmas and generate parallel code. Advanced users who prefer to use OpenMP* as it is implemented by the LLVM community can get most of that functionality by using -fopenmp.
```
and
```
  -qopenmp                Parse OpenMP pragmas and generate parallel code.
```

 Is there a reason to prefer one over the other?

Reply:
Fair enough, let's keep as is.